lib/windows/home.py

```diff
@@ -367,6 +367,12 @@ def unhookSignals(self):
         util.MONITOR.off('changed.watchstatus', self.updateOnDeckHubs)
 
     def tick(self):
+        winID = xbmcgui.getCurrentWindowId()
+        if winID < 13000 and winID not in (12901, 12005, 12002, 12000):
```

Comment:
This should probably be  `winID < 13000 or winID > 13099'.

Also I think we might need some more exceptions, such as 10123 (osdvideosettings) and 10124 (osdaudiosettings) and any dialog that could conceivably (and should be allowed to) open on top of the addon. One thing we definitely don't want is the add-on closing suddenly when it shouldn't :smile:. Instead of exceptions, it might actually make sense to check to see if the window is one that is likely to be activated from outside the addon - though I'm not sure on that, just a thought.

Reply:
Yeah you know more about the important WindowIDs - I was expecting a line-comment here. Perhaps you could change that condition properly?